www/js/lib/uiUtil.js

```diff
@@ -244,6 +252,7 @@ define([], function() {
         displayActiveContentWarning: displayActiveContentWarning,
         displayFileDownloadAlert: displayFileDownloadAlert,
         isElementInView: isElementInView,
-        htmlEscapeChars : htmlEscapeChars
+        htmlEscapeChars : htmlEscapeChars,
+        removeAnimationClasses : removeAnimationClasses,
```

Comment:
@vasusharma7 The last exported function shouldn't have a comma. But as you'll probably add the `addAnimation` function here, you can just add that one after this (with no comma at the end of addAnimation).

Reply:
Ok i will make this sure